tests/internal/test_forksafe.py

```diff
@@ -288,61 +288,91 @@ def fn():
     assert exit_code == 42
 
 
-# FIXME: subprocess marks do not respect pytest.mark.skips
-if sys.version_info < (3, 11, 0):
-
-    @pytest.mark.subprocess(
-        out=("CTCTCT" if sys.platform == "darwin" or (3,) < sys.version_info < (3, 7) else "CCCTTT"),
-        err=None,
-        env=dict(_DD_TRACE_GEVENT_HUB_PATCHED="true"),
-    )
-    def test_gevent_reinit_patch():
-        import os
-        import sys
+@pytest.mark.subprocess(
+    out=("CTCTCT" if sys.platform == "darwin" or (3,) < sys.version_info < (3, 7) else "CCCTTT"), err=None
+)
+def test_gevent_gunicorn_behaviour():
```

Comment:
I vaguely recall a thread where @P403n1x87 and I discussed if we need this test case anymore since we have the gunicorn test suite. @emmettbutler did we resolve that?

Reply:
https://github.com/DataDog/dd-trace-py/pull/4863#discussion_r1102800863 I like keeping it